src/sass/index.sass

```diff
@@ -176,8 +185,8 @@
               white-space: unset
 
             .item-summary
-              flex: unset
-
+              flex: 0 0 100%
+              line-height: 16px
```

Comment:
Paragraphs in item descriptions had 30px line-height which made the line breaks huge, this sets the line height back to the default